### DISCUSSION TITLE
[Feature]: Markdown 3.4.4 support

### DISCUSSION THREAD

**Discussion by orcephrye** [django-wiki/django-wiki#1312] [Ideas]
### I've already checked other feature proposals

Yes

### Is your feature request related to a problem?

Yes

### Describe your problem.

We should upgrade to Markdown 3.4.4 before adding additional extension support. As per https://github.com/django-wiki/django-wiki/issues/1197

Also, Markdown 3.4.4 adds support for adjusting the div for TOC which is a feature I am looking for.

### Describe your feature request.

I have made a branch already with some changes that appear to be working and passing tests on Python 3.8. 

https://github.com/orcephrye/django-wiki/tree/python_markdown_34_support

### Anything else we should know.

Help is super appreciated here. I feel a little out of my depth when making sure that there haven't been any changes under the hood that a test doesn't look for. 

Also, perhaps we should remove the tests for Python 3.7 as it has recently been deprecated.

**Comment by benjaoming** [comment]
This sounds great! Can you open a PR, and we can run the test suite? :+1:

**Comment by oscarmcm** [comment]
I started working on this here https://github.com/django-wiki/django-wiki/pull/1274

I believe I still have a stash on my local with a bunch of fixes, feel free to open a new PR with your changes so I can help there.

> we should remove the tests for Python 3.7 as it has recently been deprecated.

Yes, I think we'll update the matrix once I finally test https://github.com/django-wiki/django-wiki/pull/1311 probably by the end of this week

**Comment by orcephrye** [comment]
Sorry for the delay. I will open a PR shortly. Got a 8 week old and my time on fun projects is sporadic at best.

**Reply by orcephrye** [reply]
Ok I have made a PR: https://github.com/django-wiki/django-wiki/pull/1313

**Reply by oscarmcm** [reply]
Awesome! I'll take a look!
